components/frontapp/actions/update-conversation/update-conversation.mjs

```diff
@@ -1,23 +1,24 @@
-// legacy_hash_id: a_A6i7wv
-import { axios } from "@pipedream/platform";
+import utils from "../../common/utils.mjs";
+import frontApp from "../../frontapp.app.mjs";
 
 export default {
   key: "frontapp-update-conversation",
   name: "Update Conversation",
-  description: "Updates a conversation",
-  version: "0.1.2",
+  description: "Updates a conversation. [See the docs here](https://dev.frontapp.com/reference/patch_conversations-conversation-id).",
+  version: "0.1.3",
   type: "action",
   props: {
-    frontapp: {
-      type: "app",
-      app: "frontapp",
+    frontApp,
+    conversationId: {
+      type: "string",
+      description: "Conversation unique identifier",
     },
-    assignee_id: {
+    assigneeId: {
```

Comment:
Can this support `async options` and maybe adding a `null` to the end of the list to unassign?